photon-lib/src/main/scala/com/linkedin/photon/ml/normalization/NormalizationContext.scala

```diff
@@ -126,6 +126,36 @@ protected[ml] class NormalizationContext(
 
       outputCoef
     }
+
+  /**
+   * Transform the variance of the model coefficients of the original space to the transformed space. Will be used in
+   * constructing the prior.
+   *
+   * w' = w ./ factor
+   * b' = w^T^ shift + b
```

Comment:
Remove the extra "^".

Reply:
I saw in all previous description we use `w^T^` like line 106 and line 72, so I assume `w^T^` is `w^T`.